### PR TITLE
SecurityProfile parameter place changes

### PR DESCRIPTION
**Pull request by jaisrajms**
We have changed place and parameter names for Security Profile. This Change will make sure that SDK is as per new change.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@jaisrajms__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (rjaiswal). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by shefaliv** [comment]
Approved.

File: src/ResourceManagement/HDInsight/HDInsight/Generated/Models/SecurityProfile.cs
Comment (cormacpayne) [inline src/ResourceManagement/HDInsight/HDInsight/Generated/Models/SecurityProfile.cs]: @jaisrajms this is a breaking change, please bump the major version

File: src/ResourceManagement/HDInsight/HDInsight/Generated/Models/SecurityProfile.cs
Comment (cormacpayne) [inline src/ResourceManagement/HDInsight/HDInsight/Generated/Models/SecurityProfile.cs]: @jaisrajms this is a breaking change

File: src/ResourceManagement/HDInsight/HDInsight/Generated/Models/SecurityProfile.cs
Comment (cormacpayne) [inline src/ResourceManagement/HDInsight/HDInsight/Generated/Models/SecurityProfile.cs]: @jaisrajms this is a breaking change

File: src/ResourceManagement/HDInsight/HDInsight/Generated/Models/Role.cs
Comment (cormacpayne) [inline src/ResourceManagement/HDInsight/HDInsight/Generated/Models/Role.cs]: @jaisrajms this is a breaking change

File: src/ResourceManagement/HDInsight/HDInsight/Generated/Models/ActiveDirectoryConfiguration.cs
Comment (cormacpayne) [inline src/ResourceManagement/HDInsight/HDInsight/Generated/Models/ActiveDirectoryConfiguration.cs]: @jaisrajms removing this entire file is a breaking change

**Review by cormacpayne — CHANGES_REQUESTED** [review]
@jaisrajms please bump the major version if you are going to make these breaking changes

**Comment by jaisrajms** [comment]
@cormacpayne as discussed over email. Can you please merge this PR.

**Comment by markcowl** [comment]
@jaisrajms @shefaliv Before we can merge this we need to know if the highlighted  breaking changes will affect the API surface of your PowerShell cmdlets and whether you plan to mitigate these changes in powershell.

**Comment by jaisrajms** [comment]
@maekcowl, We have plan to release PowerShell cmdlet for SecurityProfile later this month. Currently we do not accept SecurityProfile parameter in powershell, so these change will not affect powershell cmdlet. This is an optional parameter for .net sdk.

**Comment by markcowl** [comment]
@jaisrajms You have also removed proeprties from the ```Role`` model and removed the ```ActiveDirectoryConfiguration``` model.  If these are represented either in output types or parameters of poqwershell cmdlets, then you will need to mitigate these changes in powershell.  Please indicate whether these are unused in your powershell cmdlets or you will need to mitigate.

**Comment by jaisrajms** [comment]
@markcowl Powershell does not use SecurityProfile currently. So removing proeprties from the Role`` model and removed theActiveDirectoryConfiguration``` model will not create problem with powershell or other clients. Currently SecurityProfile is only used in .Net SDK.

**Comment by markcowl** [comment]
@jaisrajms SOunds good.  This has merge conflicts due to marging the previous HDInsight changes, please update (please use git pull --rebase and force push or a similar mechanismto your branch to avoid a merge commit)

**Comment by jaisrajms** [comment]
@markcowl resolved the conflict and update the tests json file as well. Please merge this PR.

**Pull request merged**
